### ISSUE TITLE
Lunr doesn't search correctly

### ISSUE THREAD

**Issue by NistorCristian** [jamalneufeld/ghostHunter#37]
Hello,

It looks like lunr v0.7.0 doesn't search correctly. For example I'm using Ghost v1.0.0 and I have multiple posts. 2 of them have the same name Welcome to Ghost. I implmented GhostHunter and it works like this:

I search for: w -> Both posts appear
I search for: we -> They don't appear
I search for: wel -> Both posts appear
...
I search for: welcome t -> Only one appears

The blogData seems to be ok but lunr does a bad parsing or something.

**Comment by Windyo** [comment]
It's actually "working as expected" — lunr creates "blobs" and searches for that. It guarantees that you will find your posts with perfect matches, but partial matches vary based on how the "blob" is created.

I think I remember you can deactivate it by passing an argument to lunr, but I forget. It should be in the lunr.js documentation.
Sent from BlueMail<http://www.bluemail.me/r?b=10256>
On Aug 8, 2017, at 12:07, NistorCristian <notifications@github.com<mailto:notifications@github.com>> wrote:

Hello,

It looks like lunr v0.7.0 doesn't search correctly. For example I'm using Ghost v1.0.0 and I have multiple posts. 2 of them have the same name Welcome to Ghost. I implmented GhostHunter and it works like this:

I search for: w -> Both posts appear
I search for: we -> They don't appear
I search for: wel -> Both posts appear
...
I search for: welcome t -> Only one appears

The blogData seems to be ok but lunr does a bad parsing or something.

**Comment by NistorCristian** [comment]
The problem for me seems to be the stopWordFilter function. If I comment all the values from the array it works ok.

**lunr.stopWordFilter is an English language stop word list filter, any words contained in the list will not be passed through the filter.**

This is why when I wrote 'a' it didn't show me any posts with 'a' inside. Same for 'the' and all the values inside that array.

Thanks for your help @Windyo ... I started looking for that variable that you told me and I found this. 👍
